content/en/docs/ambient/install/helm/all-in-one/test.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2154
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+
+set -o pipefail
+
+# @setup profile=none
+
+snip_delete_crds
+snip_delete_system_namespace
```

Comment:
(this comment was written but never submitted sorry, so my apologies if this is in fact fixed and GitHub was showing me the old revision)

Reply:
It was and it did.  Thanks <s>Obama</s> GitHub

Reply:
Yeah, when running the test locally as a one-off it keeps failing because of stale CRDs from some previous test that is ultimately skipped, I plan to remove this.